### ISSUE TITLE
Enable a Heartbeat mechanism

### ISSUE THREAD

**Issue by hrishiballal** [hrishiballal/airtraffic-data-protocol-development#11]
It would be nice to have a heartbeat mechanism for the protocol

**Comment by cmmcdermott** [comment]
The protocol does contain a heartbeat/health monitor.  It is under the guise of the status JSON.  The rate that the status JSON sent is configurable.  This has been very useful for mobile pingStations be able to send GPS location and stay updated on the maps.  Of course for fixed pingStations things are sent less frequently but still give the "I am alive" even if no aircraft are detected.

{
	"status": {
		"pingStationGuid":"754104714b10005f",
		"pingStationVersionMajor":1,
		"pingStationVersionMinor":0,
		"pingStationVersionBuild":28,
		"timeStamp":"2018-08-07T21:21:35.104Z",
		"pingStationLatDD":42.028412,
		"pingStationLonDD":-91.717712,
		"pingStationAltType":1,
		"pingStationAltMM":216958,
		"gpsStatus":3,
		"receiverStatus":0
	}
}

**Comment by hrishiballal** [comment]
@cmmcdermott Yes agreed. I think that there will be some manufacturers who might not want to provide heartbeat so this should be left to the manufacturers. Perhaps we should standardize what the heartbeat JSON should look like?
